gdcdictionary/schemas/clinical_trial.yaml

```diff
@@ -70,277 +70,333 @@ properties:
 
   submitted_date:
     description: >
-      First Submitted Date
+      The date on which the study sponsor or investigator first submitted a study record to ClinicalTrials.gov.
     type: string
 
   first_posted_date:
     description: >
-      First Posted Date
+      The date on which the study record was first available on ClinicalTrials.gov.
     type: string
 
   last_update_posted_date:
     description: >
-      Last Update Posted Date
+      The most recent date on which changes to a study record were made available on ClinicalTrials.gov.
     type: string
 
-  actual_study_start_date:
+  brief_title:
     description: >
-      Actual Study Start Date
-    type: string
-
-  actual_primary_completion_date:
-    description: >
-      Actual Primary Completion Date
-    type: string
-
-  title:
-    description: >
-      Name for this study (Brief Title)
+      A short title of the clinical study written in language intended for the lay public. The title should include, where possible, information on the participants, condition being evaluated, and intervention(s) studied.
     type: string
 
   official_title:
     description: >
-      Official name for this study (Official Title)
+      The title of the clinical study, corresponding to the title of the protocol.
     type: string
 
   description:
     description: >
-      What this is study doing (Brief Summary)
+      A short description of the clinical study, including a brief statement of the clinical study's hypothesis, written in language intended for the lay public. (Brief Summary)
```

Comment:
maybe instead of `(Brief Summary)` we can say something like `("Brief Summary" on ClinicalTrials.gov)`

Reply:
I will push another commit to address this.

Reply:
Some of the fields actually were changed when I felt that it aligned closely with FHIR or that FHIR did a better job.